app/views/posts/index.html.erb

```diff
@@ -2,7 +2,7 @@
   <div id='post-<%= post.id %>' class="post">
     <div class="post__head">
       <span class="post__head__user">
-        <%= post.id  %>
+        <%= post.user.first_name %> <%=post.user.last_name  %>
```

Comment:
Is there a way to make this code simpler? What is the object that you need for the names? How could you pass it in?